mobile/src/bundles/employer/screens/candidates-filter/candidates-filter.tsx

```diff
@@ -14,20 +19,26 @@ const CandidatesFilter: React.FC = () => {
         // TODO: navigate to Candidates page
     }, []);
 
+    const isCommonDataLoading = commonDataStatus === DataStatus.PENDING;
+
     return (
-        <ScrollView
-            style={[
-                globalStyles.defaultScreenPadding,
-                globalStyles.borderRadius10,
-                globalStyles.width100,
-                globalStyles.height100,
-            ]}
-        >
-            <CandidatesFilterForm
-                onSubmit={handleFormSubmit}
-                onFilterClose={handleFilterClose}
-            />
-        </ScrollView>
+        <>
+            <Overlay isActive={isCommonDataLoading} />
```

Comment:
loader

Reply:
I have change it to loader, but it's looks so ugly.
As for me the best way is set it in select component, it looks good, and user see that data fething.  User expirence is good and he can open/close it in any time. 
Second way is layout, worst user expirence, but looks good....

Reply:
Ok, let's keep overlay then)